.luacheckrc

```diff
@@ -1,30 +1,7 @@
-std = "lua51+minetest"
-unused_args = false
-allow_defined_top = true
-max_line_length = 999
-
-stds.minetest = {
-	read_globals = {
-		"DIR_DELIM",
-		"minetest",
-		"core",
-		"dump",
-		"vector",
-		"nodeupdate",
-		"VoxelManip",
-		"VoxelArea",
-		"PseudoRandom",
-		"ItemStack",
-		"default",
-		table = {
-			fields = {
-				"copy",
-			},
-		},
-	}
+globals = {
+    "minetest", "basic_materials",
```

Comment:
Is there actual reason to have `minetest` in globals? Should it be under `read_globals` instead?

Reply:
blame @OgelGames in [mt-mods/steel](https://github.com/mt-mods/steel/blob/master/.luacheckrc#L3) ( i used that .luacheckrc file as a basis to keep him from complaining on this subject)

Reply:
anyways, i believe it is in globals rather than read globals is because overriding the api is used in some mods, however ogelgames is the same person who complained that a boilerplate luacheck was to much only what is being used should be in it. so I am unsure.

Reply:
resolving this as its not a big deal and can be resolved with a simple commit if desired (i have no opinion either way)

Reply:
Hmm, right, it should be in `read_globals`, I've always put it in `globals`, but never really thought about why.

BTW, why is `basic_materials` in there? There shouldn't need to be a namespace for this mod... 😕

Reply:
> Hmm, right, it should be in `read_globals`, I've always put it in `globals`, but never really thought about why.
> 
> BTW, why is `basic_materials` in there? There shouldn't need to be a namespace for this mod... 😕

legacy compat